manifests/runner.pp

```diff
@@ -153,7 +153,7 @@
     }
 
     if $description {
-        $description_opt = "--description=${description}"
+        $description_opt = "--name=${description}"
```

Comment:
This is version dependent - it should switch on the version number (as do things like the config file location).